std/string.d

```diff
@@ -5847,21 +5848,21 @@ bool isNumeric(const(char)[] s, in bool bAllowSep = false) @safe pure
         }
 
         // Allow only one exponent per number
-        if (c.among!('e', 'E')())
+        if (c == 'e' || c == 'E')
```

Comment:
I don't see why this is a negative change. IMO `among` is overkill here and with DMD's infamous inlining this will probably be faster in most cases.